.github/workflows/long_faucet_chain_test.yml

```diff
@@ -0,0 +1,37 @@
+name: Long Faucet chain test
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
This means that the test is run every time a commit is pushed on main. That may still be a lot.

Reply:
No that's fine